packages/web-components/src/select/select.spec.md

```diff
@@ -0,0 +1,100 @@
+# Select
+
+## Component Description
+
+The browser-native select control component allows users to choose one option from a dropdown list. It is commonly used in forms for selecting values from a predefined set of options.
+
+## Design Spec
+
+[Link to Design Spec in Figma](https://www.figma.com/file/0vOH481cv1VQyfQIX9JALn/Select?node-id=1319-163&t=nn2xgT4ieHjbkqQW-11)
+
+## Engineering Spec
+
+### Inputs
+
+- @attr appearance: "outline" "underline" "filled-darker" "filled-lighter" | "outline"
+- @attr size: "small" "medium" "large" | "medium"
+- @attr disabled: boolean
+- @attr multiple: boolean
+- @attr required: boolean
+- @attr name: string
```

Comment:
Are form and size necessary to pass here as well? With that in mind, what does form association look like for this given the shadow DOM? I have a feeling we'll need to recreate any events that are natively emitted from the select.

Reply:
I believe size is necessary as the Fluent React select has a size attribute but it doesn't align to the native `size` element from the browser, which corresponds to the number of visible rows when using the `multiple` attribute. But since we're not using `multiple`, can we keep `size` as an attribute that reflects the same intention as the Fluent React version?

Reply:
Well, `size` is different here in both cases. So, I don't think you can have conflicting attributes - one would need to be different. So, if `size` as it's detailed in the MDN docs is not supported because multiple is not, we should call that out. We should probably call out that multiple is also not suppoprted.

Reply:
@miroslavstastny Do we have any other recommendations or precedent for the word `size` if the `size` attribute means something else? `component-size`? `height`? Open to suggestions.

Reply:
Given multiple isn't supported here, this is probably primarily a problem for at least "Dropdown" and possibly "Combobox"

Reply:
Are you implying that it's acceptable to use `size` here as an attribute since we're not supporting `multiple`? I'm cool either way, just want to make sure we're on the same page here. A good practice might be to mimic whatever we're using for Dropdown here. I know we're using size elsewhere as a way to change the size, e.g. Slider.

Reply:
The opposite - size isn't needed here but the question for @miroslavstastny may be pertinent where multiple *is* suported.

Reply:
I got lost in the discussion here, sorry.
FUIR9 supports `size="small"|"medium"|"large"` which affects font, height and spacing:
https://github.com/microsoft/fluentui/blob/5745de0ab6ab860f905c33d4430b857b8a72f27f/packages/react-components/react-select/src/components/Select/useSelectStyles.ts#L144-L149

So we will need to support this kind of sizing in FUIWC3 as well, right?

I would expect the attribute name to be consistent across all the three components - select, dropdown and combobox. If we cannot use `size` in all three we need to find a new consistent name. I would propose `control-size` but not sure it is any better than `component-size`.

Reply:
Issue isn't in select, but for anything with multi-select, there is a size attribute which already exists in the native API for how many are shown in the listbox.